Svg.Tests.Win/SvgVisualElementsExtensionsTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Linq;
+using NUnit.Framework;
+using Svg.Editor.Tests;
+
+namespace Svg.Tests.Win;
+
+[TestFixture]
+public class SvgVisualElementsExtensionsTests
+{
+    [SetUp]
+    public void SetUp()
+    {
+        SvgPlatform.Init();
+    }
+
+    [Test]
+    public void WhenGettingBoundsFromVisualElement_WithClipPath_GetClipPathBound()
```

Comment:
Better naming,
But this implies that _alway_ the clippath bounds should be used.
Is that the case?